<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Range</class>
 <widget class="QWidget" name="Range">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>343</width>
    <height>137</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>231</x>
     <y>10</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>L1.3 Range:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>L2.2 Range:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="L32_SB">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>68</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="value">
    <number>15</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="L22_SB">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>38</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="value">
    <number>10</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>231</x>
     <y>40</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>L2.3 Range:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="L21_SB">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>38</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="value">
    <number>10</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="L23_SB">
   <property name="geometry">
    <rect>
     <x>291</x>
     <y>38</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="value">
    <number>10</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>L2.1 Range:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>100</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="L13_SB">
   <property name="geometry">
    <rect>
     <x>291</x>
     <y>8</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="value">
    <number>10</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="L31_SB">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>68</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="value">
    <number>15</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>70</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>L3.2 Range:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>231</x>
     <y>70</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>L3.3 Range:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>L1.2 Range:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="L33_SB">
   <property name="geometry">
    <rect>
     <x>291</x>
     <y>68</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="value">
    <number>15</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="L12_SB">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>8</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="value">
    <number>10</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>L3.1 Range:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="L11_SB">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>8</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="value">
    <number>10</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>L1.1 Range:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
